src/test/javascript/spec/app/admin/user-management/user-management.component.spec.ts

```diff
@@ -28,32 +32,6 @@ describe('Component Tests', () => {
             service = fixture.debugElement.injector.get(UserService);
         });
 
-        describe('OnInit', () => {
-            it('Should call load all on init', inject(
-                [],
-                fakeAsync(() => {
-                    // GIVEN
-                    const headers = new HttpHeaders().append('link', 'link;link');
-                    spyOn(service, 'query').and.returnValue(
-                        of(
-                            new HttpResponse({
-                                body: [new User(123)],
-                                headers,
-                            }),
-                        ),
-                    );
-
-                    // WHEN
-                    comp.ngOnInit();
-                    tick(); // simulate async
-
-                    // THEN
-                    expect(service.query).toHaveBeenCalled();
-                    expect(comp.users[0]).toEqual(jasmine.objectContaining({ id: 123 }));
-                }),
-            ));
-        });
-
```

Comment:
I removed this test case as the `loadAll()` in `user-management.component.ts` is not any longer directly dependent on the `query()` method in `user.service.ts`. In this commit ngx-store was introduced: https://github.com/ls1intum/Artemis/commit/cc24454994b464fb0033bb1abb3110e577a825a3#diff-4af81ae5b1dc6f8eb6c64db0483e7951R137